variables.tf

```
@@ -51,7 +51,7 @@ variable "github_repositories" {
     // organization/repository format used by GitHub.
     condition = length([
       for repo in var.github_repositories : 1
-      if length(regexall("^[A-Za-z0-9_.-]+?/[A-Za-z0-9_.-]+$", repo)) > 0
+      if length(regexall("^[A-Za-z0-9_.-]+?/([A-Za-z0-9_.-]+|\*)$", repo)) > 0
```

```suggestion
      if length(regexall("^[A-Za-z0-9_.-]+?/([A-Za-z0-9_.-]+|\\*)$", repo)) > 0
```

I think the slash also needs to be escaped, I've tested the above suggestion and it appears to work for me.